exporter/datadogexporter/README.md

```diff
@@ -12,6 +12,37 @@
 
 Visit the [official documentation](https://docs.datadoghq.com/tracing/trace_collection/open_standards/otel_collector_datadog_exporter/) for usage instructions.
 
+## FAQs
+
+### Why am I getting errors 413 - Request Entity Too Large, how do I fix it?
+
+This error indicates the payload size sent by the Datadog exporter exceeds the size limit (see previous examples https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16834, https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17566).
+
+This is usually caused by the pipepline batching too many telemetry data before sending to the Datadog exporter. To fix that, try lowering `send_batch_size` and `send_batch_max_size` in your batchprocessor config. You might want to have a separate batch processor dedicated for datadog exporter if other exporters expect a larger batch size, e.g.
+```
+processors:
+  batch:  # To be used by other exporters
+    timeout: 1s
+    # Default value for send_batch_size is 8192
+  batch/2:
+    send_batch_max_size: 100
+    send_batch_size: 10
+    timeout: 10s
+...
+service:
+  pipelines:
+    metrics:
+      receivers: ...
+      processors: [batch/2]
+      exporters: [datadog]
+```
+
+The exact values for `send_batch_size` and `send_batch_max_size` depends on your specific workload. Also note that, Datadog intake has different payload size limits for the 3 singal types:
+- 3.2MB for Trace intake
+- 1MB for Log intake
+- 500KB uncompressed, or 5MB after decompression for Metrics V2 intake
```

Comment:
Do we have a public docs reference for this? It may be better to link that instead of putting the numbers here, to avoid this getting outdated

Reply:
I found the reference for Log and Metric, but wasn't able to find that for Trace. Do you happen to know where the 3.2MB limit for Trace comes from?

Reply:
I have not been able to find a reference for this either on our public documentation. @gbbr do you know if we have such reference?